loom/engine/loom2d/l2dMatrix.h

```diff
@@ -222,16 +222,21 @@ class Matrix
 
     inline void invert()
     {
-        float a  = this->a;
-        float b  = this->b;
-        float c  = this->c;
-        float d  = this->d;
-        float tx = this->tx;
-        float ty = this->ty;
+        invertOther(this);
+    }
+
+    inline void invertOther(const Matrix *other)
```

Comment:
Needed non this-based invert for other native code.

On Tue, 02 Jun 2015, 19:42 Ben Garney <notifications@github.com> wrote:

> In loom/engine/loom2d/l2dMatrix.h
> <https://github.com/LoomSDK/LoomSDK/pull/322#discussion_r31549614>:
>
> > @@ -222,16 +222,21 @@ class Matrix
> >
> >      inline void invert()
> >      {
> > -        float a  = this->a;
> > -        float b  = this->b;
> > -        float c  = this->c;
> > -        float d  = this->d;
> > -        float tx = this->tx;
> > -        float ty = this->ty;
> > +        invertOther(this);
> > +    }
> > +
> > +    inline void invertOther(const Matrix *other)
>
> What does this gain us? Is this a Luke-ism?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LoomSDK/LoomSDK/pull/322/files#r31549614>.
>